### DISCUSSION TITLE
开始抓包提示抓包模块出错

### DISCUSSION THREAD

**Discussion by wuchen160** [sunfkny/genshin-gacha-export#35] [提问 / Q&A] [answered]
正在捕获链接...抓包模块出错:
 Traceback (most recent call last):
  File "main.py", line 442, in <module>
  File "main.py", line 286, in capture
ModuleNotFoundError: No module named 'mitmproxy.proxy.config'
 系统代理手动设置127.0.0.1:8889也无法正常使用

**Comment by sunfkny** [comment, marked as answer]
https://github.com/sunfkny/genshin-gacha-export/actions/runs/1703840073
试一下这个，7.0版本抓包模块有更改，我没测试，而且后面也打算去掉抓包功能了
